vscode-extension/src/aiConfigEditor.ts

```diff
@@ -144,23 +144,58 @@ export class AIConfigEditorProvider implements vscode.CustomTextEditorProvider {
             return;
           }
 
+          // TODO: Should we skip events with no content changes?
+          // if (e.contentChanges.length === 0) {
+          //   console.log(
+          //     `changeDocumentSubscription ${e.document.uri} - skipping event because there are no content changes`

```

Comment:
In general, would this case even be possible? Why would `changeDocumentSubscription` change if the content hasn't been changed: https://lastmileai.dev/workbooks/clsf1z185004bqpof6z9lbpvn

Reply:
Yep, it's possible and 100% happens every time you close the editor with unsaved changes. It also seems to happen pretty regularly when just making normal changes in the editor

Reply:
Ok then yea, think it would be good just to ignore if there are no contentChanges detected, as long as the "Are you sure you want to remove unsaved changes" dialog still pops up